classes/spec_metadata_generator.rb

```diff
@@ -24,6 +24,14 @@ def save
     File.open(json_filepath, "wb") { |f| f.write hash_string }
   end
 
+  def latest_version
+    versions.reverse_each.find { |v| !v.prerelease? } || versions.last
```

Comment:
I assume the versions are correctly sorted? (I can see the code on line 13, but am not 100% sure I get that guarantee)